styles/widgets/material/fileUploader.material.less

```diff
@@ -86,6 +88,33 @@
     .dx-fileuploader-button .dx-button-content {
         padding: 0;
     }
+
+    .dx-fileuploader-file-container {
+        .box-shadow(0 1px 3px 0px rgba(0, 0, 0, 0.1));
+        padding: 5px 8px;
+        margin-bottom: 4px;
+
+        .dx-fileuploader-button-container + .dx-fileuploader-button-container {
+            .dx-button {
+                margin-right: 15px;
+
+                .dx-rtl & {
+                    margin-left: 15px;
+                    margin-right: 0;
+                }
+            }
+        }
+
+        .dx-button {
+            height: 36px;
```

Comment:
I think it is better to add "width: 36px" option here

Reply:
I've fixed it

Reply:
I've fixed it